editor/import_dock.cpp

```diff
@@ -531,10 +552,27 @@ ImportDock::ImportDock() {
 	import->set_text(TTR("Reimport"));
 	import->set_disabled(true);
 	import->connect("pressed", callable_mp(this, &ImportDock::_reimport_attempt));
+	if (!DisplayServer::get_singleton()->get_swap_cancel_ok()) {
+		advanced_spacer = hb->add_spacer();
+		advanced = memnew(Button);
+		advanced->set_text(TTR("Advanced.."));
```

Comment:
Use 3-dot ellipsis for buttons that show a dialog:

```suggestion
		advanced->set_text(TTR("Advanced..."));
```